docs/tutorials/functions.ipynb

```diff
@@ -48,8 +48,8 @@
     "def kindergeld():\n",
     "    pass\n",
     "\n",
-    "user_functions = kindergeld\n",
-    "user_functions"
+    "functions = kindergeld\n",
```

Comment:
The examples do not have much connection to real-world examples due to the author's lack of knowledge 😅.

But, we have still a system of functions and not a single function, since the passed function is added to the collection of internal functions.

Reply:
I guess that I am just confused what you want to convey in the notebook then. 

If you write:

> One way to alter the tax and transfer system is to pass a function.

I presume there already is a system present. So there should be, at least per the new design, first a call to `set_up_policy_environment`. Then you alter something. This way I do not know what you are talking about :wink:

Reply:
You are right. Then, should I add more context by using `set_up...` and `compute_...` or say this is purely about how functions are loaded?

Reply:
oops, a message got lost somewhere. `set_up_...` yes, only mention `compute_...`. Or, given that this is directed at advanced users, show how the DAG is built up then. 

On another node, do we want to push the differentiation between dict key and function name? If so, we should look for an example where it makes a whole lot of sense.

Reply:
To the second point, it is definitely an advanced feature, but I think it makes a lot of sense. For example, if you have isolated your policy reform to one function which is say `kindergeld` and you have multiple proposals (functions), then you could easily loop over all functions, set them in a dictionary `{"kindergeld": func}` and apply it without altering related functions. Furthermore, you could give your functions more expressive names and keep them in the same module.

Reply:
Cool, but then I would expect the example exactly the other way around. That is, I want to replace current `kindergeld_m` by three alternatives. I would call them in three different ways and set `{"kindergeld_m": kindergeld_m_xyz}` and so on, right?

Reply:
Yes, you are right. I will overhaul it today.